rekor_common.proto

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.rekor.v2.common;
+
+option go_package = "github.com/sigstore/rekor/pkg/generated/protobuf/common";
+option java_package = "dev.sigstore.rekor.v2.common";
+option java_multiple_files = true;
+option java_outer_classname = "RekorCommonProto";
+
+message Signature {
+    /*
+     * The crypto algorithm used to generate the signature
```

Comment:
nit: This sounds like the signature algorithm, i.e rsa-pss, ecdsa, whereas it's actually the format to hold the signature. Consider rephrasing

Reply:
yeah, this is a carry forward from the current implementation... open to alternative suggestions for sure